php/deprecated.sls

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+php-deprecated-in-v1.0.0-test-fail:
+  test.fail_without_changes:
```

Comment:
```suggestion
  test.succeed_without_changes:
```

* Since we're only warning and not preventing, we want the state to pass.
* Once the deprecation takes place, we'll change this back.